packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -371,6 +371,16 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
 
   Set<SemanticsAction> _semanticActions;
 
+  /// Called whenever the scroll position or the dimensions of the scroll view
+  /// change to update the available semantics actions.
```

Comment:
this only schedules a frame to update the semantics tree, right? as opposed to updating it synchronously.

Reply:
Yes, clarified.